config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml

```diff
@@ -5,7 +5,7 @@ presets:
   # Note: The GCE Node image used may have a dependency on the nvidia-driver-installer image defined in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml
   # If updating the image defined here, the cos-gpu-installer image may need to updated to support the corresponding COS image.
   - name: KUBE_GCE_NODE_IMAGE
-    value: cos-85-13310-1041-9
+    value: cos-97-16919-103-16
```

Comment:
Can you use `cos-97-16919-189-5` here? It is the latest version as per https://cloud.google.com/container-optimized-os/docs/release-notes/m97


This image doesn't seem to exist:

```
$ gcloud compute images list --project="cos-cloud" --no-standard-images | grep -c "cos-97-16919-103-16"
0
```

Reply:
Done

Reply:
```
[davanum@c889f3bd53ed 14:24] ~/go/src/k8s.io/test-infra ⟩ gcloud compute images list --project="cos-cloud" --no-standard-images | grep "cos-97-16919-189-5"
cos-97-16919-189-5                cos-cloud  cos-97-lts                     READY
```